lib/notification_import/seed_notification_types.rb

```diff
@@ -0,0 +1,21 @@
+module NotificationImport
+  class SeedNotificationTypes
+    def initialize
+      @init_types = %w[Mentions Replies Likes Follows Posts]
+    end
+
+    def create_notification_types!
+      @init_types.each do |type|
+        NotificationSetting.where(
+          setting_name: type.titleize
+        ).first_or_create
+      end
+    end
+
+    def run!
+      ActiveRecord::Base.logger = Logger.new(nil)
+      Chewy.strategy(:bypass)
+      create_notification_types!
+    end
+  end
+end
```

Comment:
The reason I used this is so that if we decided to add more attributes in the future we can keep in the database and allow for what kinds of media we wanted to allow this for.